README.md

```diff
@@ -97,6 +97,17 @@ foo'` to search for '#define foo'. See [issue #5].
 
 ## Possibly FAQ
 
+
+#### Can I use `rg` ([Ripgrep]) with this?
+
+Absolutely, You can write a pattern match for it--just set `g:ackprg`. If you like, you can fall back to Ack in case you use your vimrc on a system without `rg` available:
+
+```vim
+if executable('rg')
+  let g:ackprg = 'rg --vimgrep'
+endif
+```
+
 #### Can I use `ag` ([The Silver Searcher]) with this?
```

Comment:
Perhaps just change this heading to something like:

    #### Can I use `ag` ([The Silver Searcher]), `rg` ([Ripgrep]) or similar with this?